app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+                        <tr>
+                          <th scope="row">Comm:</th>
+                          <td><%= link_to contact.claims_tel_comm, "tel:" + contact.claims_tel_comm %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_tel_tollfree %>
+                        <tr>
+                          <th scope="row">Toll-Free:</th>
+                          <td><%= link_to contact.claims_tel_tollfree, "tel:" + contact.claims_tel_tollfree %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_email %>
+                        <tr>
+                          <th scope="row">Email:</th>
+                          <td><%= link_to contact.claims_email.sub(',', '<br>').html_safe, "mailto:" + contact.claims_email %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_dsn %>
+                        <tr>
+                          <th scope="row">DSN Fax:</th>
+                          <td><%= contact.claims_fax_dsn %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_comm %>
+                        <tr>
+                          <th scope="row">Comm Fax:</th>
+                          <td><%= contact.claims_fax_comm %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_fax_tollfree %>
+                        <tr>
+                          <th scope="row">Toll-Free Fax:</th>
+                          <td><%= contact.claims_fax_tollfree %></td>
+                        </tr>
+                      <% end %>
+                      <% if contact.claims_post %>
+                        <tr>
+                          <th scope="row">Address:</th>
+                          <td><%= simple_format(contact.claims_post) %></td>
+                        </tr>
+                      <% end %>
+                    </tbody>
+                  </table>
+                </td>
+              </tr>
+            <%- end -%>
+          </tbody>
+        </table>
+
+        Contact for:
+        <ul>
+        	<li>Lost or Damaged Goods</li>
+          <li>Claims Negotiations with Transporter</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="usa-grid-full customer-service-section" id="retirement-separation">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("retirement-separation.png") %>
+        <h4>Retirement / Separation Extensions</h4>
+      </div>
+      <div class="usa-width-three-fourths">
+        <table>
+          <tbody>
+            <%- @contacts.each do |contact| -%>
+              <% if contact.retiree_dsn || contact.retiree_tel_comm || contact.retiree_tel_tollfree || contact.retiree_email || contact.retiree_fax_dsn || contact.retiree_fax_comm || contact.retiree_fax_tollfree || contact.retiree_post %>
```

Comment:
This could be moved out of the view and into a method on the model:

```rb
class BranchOfServiceContact < ApplicationRecord
  validates :branch, presence: true
  
  def retiree_contact_info?
    retiree_dsn || 
    retiree_tel_comm || 
    retiree_tel_tollfree || 
    retiree_email || 
    retiree_fax_dsn || 
    retiree_fax_comm || 
    retiree_fax_tollfree || 
    retiree_post
  end
end
```

You could then simplify this line to:

```erb
<%- if contact.retiree_contact_info? -%>
  …
<%- end -%>
```